frontend/public/locales/en/common.json

```diff
@@ -2,14 +2,14 @@
   "Index": {
     "page_title": "Home | Beta.grants.gov",
     "meta_description": "A one‑stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.",
-    "alert_title": "This website is a work in progress.",
-    "alert": "To search for funding opportunities and apply, visit <LinkToGrants>www.grants.gov</LinkToGrants>.",
+    "alert_title": "Beta.grants.gov is a work in progress.",
```

Comment:
I chose to say "~This website~ Beta.grants.gov is a work in progress." as it's more direct. And I'm now wondering if the "Grants.gov" in the site header should be changed to include "beta" too.